spaceship/static/wip-overlay.js

```diff
@@ -0,0 +1,10 @@
+(function() {
+  window.addEventListener('load', () => {
```

Comment:
$(window).load(function() { ... }); is idiomatic for jquery.

Reply:
Looks like $.load is deprecated, but I've changed it to use $.on